images/linux/scripts/installers/kubernetes-tools.sh

```diff
@@ -24,7 +24,9 @@ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-
 sudo install minikube-linux-amd64 /usr/local/bin/minikube
 
 # Install kustomize
-sudo snap install kustomize
+curl -s "https://raw.githubusercontent.com/\
```

Comment:
Could you please move url to the separate variable for readability?
```
download_url="https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
curl -s $download_url | bash
mv kustomize /usr/local/bin
```

Reply:
done